sdk/nodejs/apiextensions/CustomResource.ts

```diff
@@ -12,10 +12,12 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-import * as pulumi from "@pulumi/pulumi"
+import * as pulumi from "@pulumi/pulumi";
 import * as inputApi from "../types/input";
 import * as outputApi from "../types/output";
 
+import * as v1beta1 from "./v1beta1";
```

Comment:
Might want to leave a TODO to autogenerate this file since this ties us to a particular API version. I'm guessing we'll want to generate overloads for each version.